pkg/downloads/versions.go

```diff
@@ -213,6 +229,13 @@ func GetElasticArtifactVersion(version string) (string, error) {
 		return "", err
 	}
 
+	// The JSON looks like this:
+	// {
+	// 	"version" : "8.8.3-SNAPSHOT",
+	// 	"build_id" : "8.8.3-b1d8691a",
+	// 	"manifest_url" : "https://artifacts-snapshot.elastic.co/beats/8.8.3-b1d8691a/manifest-8.8.3-SNAPSHOT.json",
+	// 	"summary_url" : "https://artifacts-snapshot.elastic.co/beats/8.8.3-b1d8691a/summary-8.8.3-SNAPSHOT.html"
+	// }
```

Comment:
Since the response is fixed, rather than putting it in a comment you could just define it as a Go struct and use `json.Unmarshal` on it.

Something like:
```go
type ArtifactsSnapshotResponse struct {
    Version string `json:"version"`
    BuildID string `json:"build_id"`
    ManifestURL string `json:"manifest_url"`
    SummaryURL string `json:"summary_url"`
}
```

Followed by

```go
import "encoding/json"

response := ArtifactsSnapshotResponse{}
err := json.Unmarshal(body, &response)
// Handle error, inspect response, etc
```

The `gabs` might have been necessary before but it doesn't seem to be now.